data/glsk/glsk-file/pom.xml

```diff
@@ -43,7 +43,7 @@
         </dependency>
         <dependency>
             <groupId>com.powsybl</groupId>
-            <artifactId>powsybl-sensitivity-api</artifactId>
+            <artifactId>powsybl-sensitivity-analysis-api</artifactId>
```

Comment:
glsk file should not rely on computation module no ?

Reply:
It allows to generate LinearGlsk variable... so it is needed